.travis.yml

```diff
@@ -77,7 +80,22 @@ matrix:
       services:
         - docker
       script:
-        - "./build/travis/job2_AppImage/build.sh --32bit --upload-branches $APPIMAGE_UPLOAD_BRANCHES"
+        - "[[ $APPIMAGE_BUILD_ARCHS == *i686* ]] && ./build/travis/job2_AppImage/build.sh --i686 --upload-branches $APPIMAGE_UPLOAD_BRANCHES || true"
+
+    # 4th parallel build job - portable Linux AppImage armhf build on Debian crosscompiler
+    - env: "JOB=AppImage_armhf"
+      addons:
+        apt:
+          packages:
+          - bsdtar
+          - curl
+          - zsync
+      services:
+        - docker
+      before_script:
+        - "sudo ./build/travis/job2_AppImage/set-binfmt-misc.sh"
+      script:
+        - "[[ $APPIMAGE_BUILD_ARCHS == *armhf* ]] && ./build/travis/job2_AppImage/build.sh --armhf --upload-branches $APPIMAGE_UPLOAD_BRANCHES || true"
```

Comment:
I also should mention before merge, to set Travis environment var APPIMAGE_BUILD_ARCHS to "x86-64 i686 armhf" (or some subset of that) to build for each architecture.  (Note: quotes needed, or else travis just sets APPIMAGE_BUILD_ARCHS to the first string before space is encountered.)